bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +200,27 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        if should_contain_url:
+            urls = re.findall(r'(https?://\S+)', body)
+            has_good_url = False
+            for url in urls:
+                pieces = urlparse(url)
+                netloc = pieces.netloc
+                scheme = pieces.scheme
+                path = pieces.path
+                try:
+                    assert all([scheme, netloc, path])
+                    if netloc.find('github') != -1 and path.find('issues'):
```

Comment:
Not all people use github so we should have this configurable for those who use other solutions